src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -79,44 +86,41 @@
       <TestCopyLocal Include="@(_ExistingReplacementCandidate)" />
```

Comment:
Are you referring to the removal of the TestWithLocalLibraries option?  I'm planning on doing that in a separate change after confirming with the dev team that nobody is using it.